minio_storage/storage.py

```diff
@@ -176,7 +175,7 @@ def delete(self, name: str) -> None:
                 ) from error
 
             try:
-                content_length = int(obj.getheader("Content-Length"))
+                content_length = int(obj.headers.get("Content-Length", ""))
```

Comment:
The old urllib3 `.getheader` API was apparently deprecated, and the `.getheader` type annotations were incorrect too.

The second argument ensures that if `"Content-Length"` is missing, then we call `int("")`, which raises a `ValueError`. Calling `int(None)` raises a `TypeError`.